### DISCUSSION TITLE
Create a PR when `automergeType=branch` has not succeded for some time

### DISCUSSION THREAD

**Discussion by schulzjo-tng** [renovatebot/renovate#30145] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Bitbucket on-premise

### Please tell us more about your question or problem

* We would like to use Automerge on our Bitbucket project.
* The default `automergeType=pr` is not viable for us because we require approvals for PRs, and Bitbucket neither supports self-approval nor an allow-list of users who can merge PRs without approval.
* We have allowed the technical user to commit to the branch without PRs (that is supported, at least).
* We have tried to use `automergeType=branch` which *does* work, but comes with another issue:
  * It can take weeks until a given branch is merged when there is a lot of activity on `main`, and nobody notices because Renovate only creates PRs when the pipeline fails.

Therefore, I was wondering if any of the following is possible:
* Use the `automergeType=branch` strategy, but create a PR if no merge was possible for a given timeframe (e.g. 72 hours).
* Use `automergeType=pr`, but commit the changes directly and close the PR instead of merging the PR via Bitbucket.

Thanks in advance for your help!

**Comment by rarkins** [comment]
Converted to issue: https://github.com/renovatebot/renovate/issues/30150

**Reply by schulzjo-tng** [reply, marked as answer]
I take that it is not possible at the moment and you have converted my question to a feature request? In any case, thanks for responding!
